src/Nethermind/Nethermind.Blockchain/Synchronization/SyncLimits/GethSyncLimits.cs

```diff
@@ -19,7 +19,7 @@ namespace Nethermind.Blockchain.Synchronization.SyncLimits
     public static class GethSyncLimits
     {
         public const int MaxHeaderFetch = 192; // Amount of block headers to be fetched per retrieval request
-        public const int MaxBodyFetch = 32; // Amount of block bodies to be fetched per retrieval request
+        public const int MaxBodyFetch = 128; // Amount of block bodies to be fetched per retrieval request
```

Comment:
check with Geth and this should be fine (incorrectly took the LES setting before)

Reply:
but I have seen bodies to be disconnected above 32 so need to confirm if it syncs nicely